exporter/exporterhelper/internal/request/sizer.go

```diff
@@ -50,6 +57,34 @@ func (s *SizerType) MarshalText() ([]byte, error) {
 	return []byte(s.val), nil
 }
 
+// ToUint32 returns the uint32 representation for protobuf serialization
+func (s SizerType) ToUint32() (uint32, error) {
+	switch s.val {
+	case sizerTypeBytes:
+		return SizerTypeBytesValue, nil
+	case sizerTypeItems:
+		return SizerTypeItemsValue, nil
+	case sizerTypeRequests:
+		return SizerTypeRequestsValue, nil
+	default:
+		return 0, fmt.Errorf("invalid sizer type: %s", s.val)
+	}
+}
+
+// FromUint32 creates SizerType from uint32 representation
+func FromUint32(value uint32) (SizerType, error) {
+	switch value {
+	case SizerTypeBytesValue:
+		return SizerTypeBytes, nil
+	case SizerTypeItemsValue:
+		return SizerTypeItems, nil
+	case SizerTypeRequestsValue:
+		return SizerTypeRequests, nil
+	default:
+		return SizerType{}, fmt.Errorf("invalid sizer type value: %d", value)
+	}
+}
```

Comment:
Please move these as helpers within the internal/persistenqueue package since we don't need them anywhere else.